ts-tests/__tests__/balances.test.ts

```diff
@@ -9,54 +10,86 @@ import {
 const container = new MetaDContainer();
 const TEST_ACCOUNT = '0x1111111111111111111111111111111111111111';
 
-beforeAll(async () => {
-  await container.start();
-});
+const value = '0x200'; // 512, must be higher than ExistentialDeposit
+const gasPrice = '0x3B9ACA00'; // 1000000000
 
-afterAll(async () => {
-  await container.stop();
-});
+// GENESIS_ACCOUNT_BALANCE - (21000 * gasPrice) - value;
+const expectedGenesisBalance = (
+  BigInt(GENESIS_ACCOUNT_BALANCE) -
+  BigInt(21000) * BigInt(gasPrice) -
+  BigInt(value)
+).toString();
+
+const expectedTestBalance = (Number(value) - EXISTENTIAL_DEPOSIT).toString();
+
+describe('web3', () => {
+  beforeAll(async () => {
+    await container.start();
+  });
+
+  afterAll(async () => {
+    await container.stop();
+  });
+
+  it('should genesis balance setup correctly', async () => {
+    const bal = await container.web3.eth.getBalance(GENESIS_ACCOUNT);
+    expect(bal).toStrictEqual(GENESIS_ACCOUNT_BALANCE);
+  });
+
+  it('should transfer balance', async () => {
+    const tx = await container.web3.eth.accounts.signTransaction(
+      {
+        from: GENESIS_ACCOUNT,
+        to: TEST_ACCOUNT,
+        value: value,
+        gasPrice: gasPrice,
```

Comment:
```suggestion
        value: value.toHexString(),
        gasPrice: gasPrice.toHexString(),
```

(if we use ethers BigNumber)